ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx

```diff
@@ -161,7 +161,7 @@ class ResultsExport extends React.Component<Props, State> {
         response.headers.get('content-disposition')
       ).parameters.filename
       const contentType = response.headers.get('content-type')
-      const data = await response.text()
+      const data = await response.blob()
```

Comment:
The response was converting the binary content to text which was corrupting the exported zip files.